rakelib/plugins-metadata.json

```diff
@@ -329,7 +329,7 @@
     "skip-list": true
   },
   "logstash-input-kafka": {
-    "default-plugins": true,
+    "default-plugins": false,
     "skip-list": false
```

Comment:
the skip-list is used to skip plugins when creating the version manifest file for a branch, that is then used to generate docs. once we have the integration plugin, we'll want to skip these to avoid having duplicate entries in the docs index.
so I'd suggest flipping the `skip-list` boolean for these plugins